truetype/truetype.go

```diff
@@ -26,6 +26,51 @@ import (
 // An Index is a Font's index of a rune.
 type Index uint16
 
+// A NameID represents the Name Identifiers in the name table.
```

Comment:
Also in there:

"The name identifier (nameID) codes provide a single word or number description relating to the name character string."

s/NameID/NameIDCode/?